exercises/random_number.cpp

```diff
@@ -4,3 +4,24 @@
   Output:
   The random number is: 4
 */
+#include <iostream>
+#include <cstdlib>
+#include <ctime>
+
+using namespace std;
+
+void randomGenerator(int& n) {
+    n = rand()%100+1;
+}
+
+int main() {
+
+    srand(time(0));
+
+    int a;
+    randomGenerator(a);
+
+    cout << "The random number is: " << a << endl;
```

Comment:
potresti evitare la creazione della variabile a facendo in modo di ritornare il valore rand() % 100 +1 in randomGenerator() e mettendolo qui
```c++
cout << "The random number is: " << randomGenerator() << endl;
```

Comunque, a volte, è questione di stile, anche il codice scritto così andrebbe bene, però questi esercizi consistono di più nell'usare git e rimodificare il proprio codice in base alle code-review